QSB/TimeSync/WakeUpSync.cs

```diff
@@ -9,7 +9,9 @@ namespace QSB.TimeSync
     public class WakeUpSync : NetworkBehaviour
     {
         private const float TimeThreshold = 0.5f;
-        private const float FastForwardSpeed = 10f;
+        private const float MaxFastForwardSpeed = 60f;
```

Comment:
Are we sure it's safe to play it that fast? Physics might not be calculated correctly 🤷‍♂

Reply:
> Are we sure it's safe to play it that fast? Physics might not be calculated correctly 🤷‍♂

I'm not sure :P But as far as I can tell, Unity only goes as fast as your computer can handle. Increasing the value after a certain point seems to make no difference. Even in the base game, sleeping goes up to 10x speed, but if your computer can't calculate physics that fast it just won't go that fast, it does't skip physics calculations. I'm sure it can break stuff that relies on the time step, though. But if the planets orbit correctly, I think that's good enough?